src/lib/viz/layer/Layer.ts

```diff
@@ -94,6 +107,7 @@ export class Layer extends WithEvents implements StyledLayer {
 
     if (this._deckLayer) {
       await this.replaceDeckGLLayer();
+      this.dataState = DATA_STATES.STARTING;
```

Comment:
This is applied to `setSource`. Would it be also necessary with `setStyle`?

Reply:
No, it would not because data does not change with setStyle